src/parser/warlock/affliction/modules/talents/Nightfall.tsx

```diff
@@ -1,49 +1,49 @@
 import React from 'react';
 
 import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
-import Events from 'parser/core/Events';
+import Events, { ApplyBuffEvent, RefreshBuffEvent, RemoveBuffEvent } from 'parser/core/Events';
 
 import SPELLS from 'common/SPELLS';
 
 import Statistic from 'interface/statistics/Statistic';
 import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
-import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
 
 const BUFF_DURATION = 12000;
 const BUFFER = 100;
 
 class Nightfall extends Analyzer {
   wastedProcs = 0;
-  buffApplyTimestamp = null;
+  buffApplyTimestamp: number = 0;
```

Comment:
This changes the functionality of the code, the timestamp is nullchecked on L28. For clarity (to avoid any 0 == null weirdness; I frankly down know if this works as-is) you can do
`buffApplyTimestamp: number | null = null` 

And then change back L40 as well